### ISSUE TITLE
Patch for /port/atomic_pointer.h

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/leveldb#124]
```
Sparcv9 AtomicPointer implementation
```

---

Original issue reported on code.google.com by `Alexande...@gmail.com` on 3 Oct 2012 at 8:56

Attachments:
- [atomic_pointer.h.patch](https://storage.googleapis.com/google-code-attachments/leveldb/issue-124/comment-0/atomic_pointer.h.patch)

**Comment by GoogleCodeExporter** [comment]
```
Patch modified: added "#undef ARCH_CPU_SPARCV9_FAMILY" at the end of the header
```

---

Original comment by `Alexande...@gmail.com` on 9 Oct 2012 at 1:59

Attachments:
- [atomic_pointer.h.patch](https://storage.googleapis.com/google-code-attachments/leveldb/issue-124/comment-1/atomic_pointer.h.patch)

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 1.6.0.

```

---

Original comment by `san...@google.com` on 12 Oct 2012 at 7:53
- Changed state: **Fixed**
